src/expandableCalendar/agendaList.js

```diff
@@ -25,13 +24,17 @@ class AgendaList extends Component {
     /** day format in section title. Formatting values: http://arshaw.com/xdate/#Formatting */
     dayFormat: PropTypes.string,
     /** style passed to the section view */
-    sectionStyle: PropTypes.oneOfType([PropTypes.object, PropTypes.number, PropTypes.array])
-  }
+    sectionStyle: PropTypes.oneOfType([
```

Comment:
Please refrain from formatting the file using your own tools and preferences, we are trying to keep the code unified.